backend/logic/printful/sync/stockUpdate.js

```diff
@@ -0,0 +1,66 @@
+const _get = require('lodash/get')
+const program = require('commander')
+const { decryptConfig } = require('../../../utils/encryptedConfig')
+
+const { get } = require('../api')
+const { Product } = require('../../../models')
+
+/**
+ * Updates discontinued variants and variants that are out of stock to a collection.
+ * And creates collection if it doesn't exist
+ *
+ * @param {Object} shop
+ * @param {Number|String} productId
+ * @param {Array<Object>} variants
```

Comment:
It would be helpful to document that variants is the list of variants that are out of stock or discontinued.

Reply:
Updated